optuna/cli.py

```diff
@@ -35,6 +40,28 @@
 
 _DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
+# The following code is quoted from the source code of SQLAlchemy project (MIT License).
+# See LICENSE file for detials.
+# https://github.com/zzzeek/sqlalchemy/blob/c6554ac52/lib/sqlalchemy/engine/url.py#L234-L292
```

Comment:
The description of the repository recommends 
 https://github.com/sqlalchemy/sqlalchemy. Do we stick to `https://github.com/zzzeek/sqlalchemy`?

![image](https://github.com/optuna/optuna/assets/3255979/ca15731a-a8c9-47d1-bc16-f0f5cb7ded58)

Reply:
Thank you for pointing it out. Good catch. We should refer the `sqlalchemy/sqlalchemy` repo.

```suggestion
# https://github.com/sqlalchemy/sqlalchemy/blob/c6554ac52/lib/sqlalchemy/engine/url.py#L234-L252
```